stories/Components/Carousels/FluidImageSizeCarousel/fluidimagesizecarousel.scss

```diff
@@ -0,0 +1,107 @@
+@import '../../../assets/scss/variables';
+@import '../../../assets/scss/mixins';
+
+.fluid-image-size-carousel {
+  box-sizing: border-box;
+  position: relative;
+
+  &__image {
+    width: 100%;
+
+    @include devicebreak(small) {
+      max-width: 10.625rem;
+    }
+  }
+
+  &__contents {
+
+    &-tag {
+      color: $color-black;
+      font-size: 0.75rem;
+      font-weight: 700;
+      line-height: 0.844rem;
+      margin-bottom: 1rem;
+      text-transform: uppercase;
+    }
+
+    h3 {
+      font-weight: 400;
+      margin-bottom: 1rem;
+    }
+
+    p {
+      font-size: 1rem;
+      line-height: 1.375rem;
+      margin-bottom: 1.5rem;
+
+      @include devicebreak(large) {
+        margin-bottom: 3rem;
+      }
+    }
+
+    .cta__link {
+      text-transform: uppercase;
+    }
+
+    @include devicebreak(small) {
+      margin-top: 1.5rem;
+      padding-left: 1.5rem;
+
+      h3 {
+        font-size: 1.563rem;
+        line-height: 1.784rem;
+        margin-bottom: 1rem;
+      }
+
+      p {
+        font-size: 0.875rem;
+        line-height: 1.203rem;
+      }
+    }
+  }
+
+  .glide {
+    &__track {
+      margin-right: -100%;
+    }
+
+    &__arrows {
+      bottom: -0.5rem;
+      position: absolute;
+      right: 5%;
+
+      @include devicebreak(large) {
+        display: none;
+      }
+
+      @include devicebreak(small) {
+        display: none;
+      }
+    }
+
+    &__arrow {
+      @include arrow;
+
+      img {
+        width: 0.625rem;
+      }
+    }
+  }
+}
```

Comment:
@Shamimkhan321 seems like fluidimagesize and fixedimagesize carousel have similar styling so can you please create the proper placeholder and mixins

Reply:
@Shamimkhan321 you have not fixed this issue